source/marshall_setup.py

```diff
@@ -0,0 +1,64 @@
+from source.CardGames import *
+
+marshall = Player("Marshall")
+deck = Deck()
+
+def marshall_first_turn():
+    # Waiting for deck to be divided into three groups
+    input("Select which deck to draw two cards from ['1' (1-14), '2' (15-28), '3' (29-42)] and press enter...")
+
+    for i in range(2):
+        draw_card = deck.getCard()
+        marshall.addCard(draw_card, isKnown=True)
+
+    marshall.showHand()
+
+    #Display board
+    
+    while(True):
+        guess_all = input("Would you like to guess all fugitive locations? (y/n)")
+        if guess_all.lower() == 'y' or guess_all.lower() == 'yes':
```

Comment:
Great work Sean! The only stylistic change I would put is to use the lower() method on the input itself; then you won't have to use the lower method on the guess_all variable multiple times throughout the loop.

Reply:
That's a great point, thanks big dawg.